docs/build/build-node-interaction.md

```diff
@@ -414,3 +414,11 @@ return an error report, e.g.:
     "cause": "Upstream error description"
 }
 ```
+
+### RPC Node APIs
```

Comment:
I would suggest adding the table before the **Polkadot RPC** subheading. 
We can also add a brief explanation for each endpoint, similar to the description in *kusama-endpoints.md* for `wss://kusama-rpc.polkadot.io/`.

Reply:
done, please review this update. thanks!

Reply:
Let's also run it by @joepetrowski